Impossible to generate Nuxt project

```shell
~/projects/o1/zkapp-cli   e2e-tests ±  zk project  deploy-me
✔ Create an accompanying UI project too? · nuxt
Nuxi 3.4.0                                                                                                                                  7:04:11 PM
✨ Nuxt project is created with v3 template. Next steps:                                                                                    7:04:12 PM
 › cd ui                                                                                                                                    7:04:12 PM
 › Install dependencies with npm install or yarn install or pnpm install                                                                    7:04:12 PM
 › Start development server with npm run dev or yarn dev or pnpm run dev                                                                    7:04:12 PM
✔ UI: Set up project
✖ UI: NPM install
 ✘  ~/projects/o1/zkapp-cli   e2e-tests ±  cd deploy-me/
 ~/projects/o1/zkapp-cli/deploy-me   e2e-tests ±  ll
total 0
drwxr-xr-x   3 serhii  staff   96 Jul  7 19:04 .
drwxr-xr-x  29 serhii  staff  928 Jul  7 19:04 ..
drwxrwxr-x  16 serhii  staff  512 Jul  7 19:04 ui
 ~/projects/o1/zkapp-cli/deploy-me   e2e-tests ±  cd ui
 ~/projects/o1/zkapp-cli/deploy-me/ui   e2e-tests ±  npm install

> postinstall
> nuxt prepare

Nuxi 3.6.2                                                                                                                                  7:05:16 PM

 ERROR  ParseError: Unexpected token, expected ","                                                                                          7:05:17 PM
 /Users/shymkiv/projects/o1/zkapp-cli/deploy-me/ui/nuxt.config.ts:12:2

  nuxt.config.ts:12:2
  at resolveConfig (node_modules/c12/dist/index.mjs:280:24)
  at loadConfig (node_modules/c12/dist/index.mjs:116:40)
  at async loadNuxtConfig (node_modules/@nuxt/kit/dist/index.mjs:2483:18)
  at async loadNuxt (node_modules/nuxt/dist/index.mjs:3269:19)
  at async loadNuxt (node_modules/@nuxt/kit/dist/index.mjs:2537:19)
  at async Object.invoke (node_modules/nuxi/dist/chunks/prepare.mjs:31:18)
  at async _main (node_modules/nuxi/dist/cli.mjs:49:20)

npm ERR! code 1
npm ERR! path /Users/shymkiv/projects/o1/zkapp-cli/deploy-me/ui
npm ERR! command failed
npm ERR! command sh -c nuxt prepare

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/shymkiv/.npm/_logs/2023-07-07T16_05_15_030Z-debug-0.log
 ✘  ~/projects/o1/zkapp-cli/deploy-me/ui   e2e-tests ±  zk -v
0.10.1
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.10.2] - TBA
+
+### Changed
+
+- Welcome page getting started text changed for the zkApp project generated with the `NextJS` UI framework ([#450](https://github.com/o1-labs/zkapp-cli/issues/450))
+
+## Fixed
+
+- Fix for zkApp project generation with `Nuxt` UI framework issue ([#449](https://github.com/o1-labs/zkapp-cli/issues/449))
+
 ## [0.10.1] - 2023-06-22
 
 ### Changed
@@ -16,11 +26,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Added
 
 - Third-party fee payer accounts for deployment transactions can be used across multiple projects [#424](https://github.com/o1-labs/zkapp-cli/pull/424)
-    - A new `zk config` option creates or selects a third-party fee payer account. 
+  - A new `zk config` option creates or selects a third-party fee payer account.
 
 ## Fixed
 
 - A fee payer tries to pay the fee using signature authorization (proofs are not supported). If the zkApp account is used as fee payer after deployment, the transaction was rejected because the permissions of the account would be violated, breaking all re-deployments and the interact.ts script in the template project. [#424](https://github.com/o1-labs/zkapp-cli/pull/424)
+
 ## [0.9.1] - 2023-06-17
 
 ### Changed
@@ -32,8 +43,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Release `0.9.0` [#426](https://github.com/o1-labs/zkapp-cli/pull/426)
-  - SnarkyJS minor version dependency updated in cli package.json to 0.11.*.
-  - SnarkyJS minor version peer dependency updated in template/example contract package.json to 0.11.*.
+  - SnarkyJS minor version dependency updated in cli package.json to 0.11.\*.
+  - SnarkyJS minor version peer dependency updated in template/example contract package.json to 0.11.\*.
   - Replace deprecated `Circuit.if` with `Provable.if` in the `tictactoe` example.
   - Replace deprecated `Circuit.array` with `Provable.Array` in the `sudoku` example.
 
@@ -52,17 +63,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [0.8.0] - 2023-05-03
 
 ### Changed
+
 - Release `0.8.0` [#399](https://github.com/o1-labs/zkapp-cli/pull/399)
+
   - Remove deprecated isReady and shutdown from cli, template/examples contracts, and UI scaffolds for each supported framework.
   - Add state.getAndAssertEquals() to examples and test contracts.
   - Configure UI scaffolds for NextJS, Svelte, and Nuxt to support top-level await used in the latest SnarkyJS release.
   - Exit the cli config and deploy processes on success and error.
   - Increase the template/example project jest test timeout.
- 
 
 - Update interact script config field. [#396](https://github.com/o1-labs/zkapp-cli/pull/396)
 
- - Update help text. [#392](https://github.com/o1-labs/zkapp-cli/pull/392)
+- Update help text. [#392](https://github.com/o1-labs/zkapp-cli/pull/392)
 
 ## [0.7.6] - 2023-04-25
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.10.2] - TBA
+## [0.10.2] - 2023-07-09
 
 ### Changed
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "zkapp-cli",
-  "version": "0.10.1",
+  "version": "0.10.2",
   "description": "CLI to create zkApps (zero-knowledge apps) for Mina Protocol",
   "homepage": "https://github.com/o1-labs/zkapp-cli/",
   "keywords": [

--- a/src/lib/project.js
+++ b/src/lib/project.js
@@ -709,7 +709,7 @@ function scaffoldNuxt() {
       optimizeDeps: { esbuildOptions: { target: "esnext" } },
     },
 
-    css: ['~/assets/styles/globals.css']
+    css: ['~/assets/styles/globals.css'],
   `
   );
 

--- a/src/lib/ui/next/customNextIndex.js
+++ b/src/lib/ui/next/customNextIndex.js
@@ -56,7 +56,7 @@ export default function Home() {
           </div>
           <p className={styles.start}>
             Get started by editing
-            <code className={styles.code}> src/pages/index.tsx</code>
+            <code className={styles.code}> src/pages/index.js</code> or <code className={styles.code}> src/pages/index.tsx</code>
           </p>
           <div className={styles.grid}>
             <a

